pkgs/development/python-modules/monai/default.nix

```diff
@@ -12,19 +12,19 @@
 
 buildPythonPackage rec {
   pname = "monai";
-  version = "1.0.0";
+  version = "1.1.0";
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "Project-MONAI";
     repo = "MONAI";
     rev = "refs/tags/${version}";
-    hash = "sha256-BnsgZpkXsY2l6inuzOkuvK4zJqYs2dj7jF5gKKiLnZM=";
+    hash = "sha256-XTjZhynIiFtFjJSW6rRAnpErZvf6QHkuK4e2L6l3naM=";
   };
 
   # Ninja is not detected by setuptools for some reason even though it's present:
   postPatch = ''
-    substituteInPlace "setup.cfg" --replace "ninja" ""
+    substituteInPlace "setup.cfg" --replace "    ninja" ""
```

Comment:
why is this change necessary?

Reply:
It's because they added an installation option `monai[ninja]` so `setup.cfg` got a line `ninja =` which we don't want to remove.